cg/clients/freshdesk/freshdesk_client.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import requests
+
+from cg.clients.freshdesk.constants import EndPoints
+from cg.clients.freshdesk.models import Ticket
+
+LOG = logging.getLogger(__name__)
+
+
+class FreshdeskClient:
+    """Client for communicating with the freshdesk REST API."""
+
+    def __init__(self):
+        self.api_key = None
+        self.url = None
+
+    def init_app(self, url, api_key):
```

Comment:
Would be nice with type hints.

Reply:
And what is the different use cases for the init and the init_app?

Reply:
Will add type hints! Regarding the `init_app` thingy there is a weird way we instantiate services for the API. We basically instantiate them in one file, before we read the config with env variables. Hence we can't assign them on instantiation... I might spend some time to fix it in a future PR, but this just follows what has been done for other services

Reply:
Fixed type hints in [4eb3e02](https://github.com/Clinical-Genomics/cg/pull/3317/commits/4eb3e027287cc3751ae83eed06d269ef3292cc51)